omnibus/config/software/datadog-agent.rb

```diff
@@ -238,6 +239,38 @@
     delete "#{install_dir}/uselessfile"
   end
 
+  # TODO: move this to omnibus-ruby::health-check.rb
```

Comment:
Is this todo being merged intentionally?

Reply:
I believe health-check is a post install stage in omnibus and this is a candidate for moving to that? I'm unsure the intent and can remove the comment

Reply:
Yes that's the idea, omnibus provides a step that does sanity checking on the built binaries, and it feels like this check would belong in there.
I think the intent from Nicolas was to take a shortcut by introducing the change here rather than having to modify omnibus directly